.github/workflows/doc_build.yml

```diff
@@ -1,13 +1,22 @@
 name: documentation build
 
-# This workflow runs on all pull requests because branch protection requires it.
-# If no documentation files change, the build steps are skipped to save time.
+# This workflow runs on all pull requests (build only)
```

Comment:
keep original comment, especially the important info on branch protection rules.